File: ios/BaseiOSSwitUI/BaseiOSSwitUI/Core/Domain/Model/CoinHomeModel.swift
//
//  CoinHomeModel.swift
//  BaseiOSSwitUI
//
//  Created by Ali Taha on 4.12.2023.
//

import Foundation

struct CoinHomeModel: Identifiable {
    let id: String
    var name: String = Constants.EMPTY_STRING
    var symbol: String = Constants.EMPTY_STRING
    var isActive: Bool = false
    var isNew: Bool = false
    var type: CoinType = .invalid
}

File: android/BaseAndroidXMLKtor/app/src/main/java/dev/subfly/baseandroidxmlktor/core/domain/model/CoinHomeModel.kt
package dev.subfly.baseandroidxmlktor.core.domain.model

import dev.subfly.baseandroidxmlktor.core.util.CoinType
import dev.subfly.baseandroidxmlktor.core.util.Constants

data class CoinHomeModel(
    val id: String,
    val name: String = Constants.EMPTY_STRING,
    val symbol: String = Constants.EMPTY_STRING,
    val isActive: Boolean = false,
    val isNew: Boolean = false,
    val type: CoinType = CoinType.COIN,
)
